WordPress/Jetpack/Classes/NUX/New Landing Screen/Views/JetpackLandingScreenView.swift

```diff
@@ -1,19 +1,26 @@
-
 import SwiftUI
 
 struct JetpackLandingScreenView: View {
+    /// If the view should start with the even color, set to `true`.
+    let startWithEvenColor: Bool
 
-    let viewModel: JetpackPromptsViewModel
+    private let prompts = JetpackPromptsConfiguration.Constants.basePrompts
+    private let evenColor = JetpackPromptsConfiguration.Constants.evenColor
+    private let oddColor = JetpackPromptsConfiguration.Constants.oddColor
 
-    var body: some View {
-        GeometryReader { proxy in
-            makeJetpackPromptsView(size: CGSize(width: proxy.size.width - JetpackPromptView.totalHorizontalPadding,
-                                                height: proxy.size.height))
-        }
+    func colorForIndex(_ index: Int) -> Color {
+        let colorOffset = startWithEvenColor ? 0 : 1
+        return (index + colorOffset) % 2 == 0 ? evenColor : oddColor
     }
 
-    private func makeJetpackPromptsView(size: CGSize) -> some View {
-        viewModel.configuration = JetpackPromptsConfiguration(size: size)
-        return JetpackPromptsView(viewModel: viewModel)
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading) {
+                ForEach(0..<prompts.count, id: \.hashValue) { index in
+                    JetpackPromptView(text: prompts[index], color: colorForIndex(index))
+                }
+            }
+            Spacer()
```

Comment:
👋 @Gio2018 and @salimbraksa, let me know if you have any better ideas for this!

`HStack` + `Spacer()` feels clunky to me, but I couldn't get `maxWidth` to work like I wanted. This is mostly for iPad to use the full width.

Reply:
That sounds fine to me.

Reply:
I think `HStack` + `Spacer()` is fine but here is another approach to achieve the same result:
```swift
var body: some View {
      VStack(alignment: .leading) {
            // Omitted for brevity
      }
      .frame(maxWidth: .infinity, alignment: .leading)
}
```

Reply:
Interesting @salimbraksa! I didn't have the `alignment` property set and that must've been the difference. It feels a little more natural to me to use the `.frame` modifier here, so I may update it to that. 🙇

Reply:
Updated here: 82b31d8d9491e157dc3b89f64ae8f2bae58ea3a2